README.md

```diff
@@ -1 +1,135 @@
-# math
\ No newline at end of file
+# qFALL-math
+[![made-with-rust](https://img.shields.io/badge/Made%20with-Rust-1f425f.svg)](https://www.rust-lang.org/)
+[![CI](https://github.com/qfall/math/actions/workflows/pull_request.yml/badge.svg?branch=dev)](https://github.com/qfall/math/actions/workflows/pull_request.yml)
+[![License: MPL 2.0](https://img.shields.io/badge/License-MPL_2.0-brightgreen.svg)](https://opensource.org/licenses/MPL-2.0)
+<TODO Badge for Documentation>
+<TODO Badge for Code Coverage>
+<TODO Badge for Website>
+
+This repository is being developed by the project group [qFALL - quantum resistant fast lattice library](https://cs.uni-paderborn.de/cuk/lehre/veranstaltungen/ws-2022-23/project-group-qfall) in the winter term 2022 and summer term 2023 in Paderborn.
+
+The main objective of this project is to develop a memory safe and efficient usage of
+[FLINT](https://flintlib.org/) in [Rust](https://www.rust-lang.org/). The main purpose
+is to use this library as a building block to build other projects on top of it.
+
+## Disclaimer
+As of right now we are in the development phase and interfaces might change.
+Feel free to check out the current progress, but be aware, that the content will
+change in the upcoming weeks and months. An official release will be published in the second half of 2023.
+
+## Installation
+In order to use this project one needs to have an [installation of Rust](https://www.rust-lang.org/tools/install). Since we are using flint-sys
+which itself uses [gmp](https://gmplib.org/manual/), we are currently restricted to usage on Mac, Linux and Linux subsystems under Windows. For a subsystem under Windows, one additionally is required to have installed m4 and a C-compiler.
+
+Since our project isn't yet published there is no option to find it on crates.io.
```

Comment:
```suggestion
Since our project isn't yet published there is no option to find it on Rust's library collection on [crates.io](https://crates.io/).
```